packages/components/tooltip/src/tooltip.vue

```diff
@@ -80,22 +80,10 @@ const emit = defineEmits(tooltipEmits)
 
 usePopperContainer()
 const compatShowAfter = computed(() => {
-  if (!isUndefined(props.openDelay)) {
-    debugWarn(
-      'ElTooltip',
-      'open-delay is about to be deprecated in the next major version, please use `show-after` instead'
-    )
-  }
-  return props.openDelay || (props.showAfter as number)
+  return props.showAfter
```

Comment:
Seems like there is no need to wrap in computed.

Reply:
You right.